fuzz/fuzz_targets/differential_spec.rs

```diff
@@ -0,0 +1,57 @@
+#![no_main]
+
+use libfuzzer_sys::arbitrary::{Result, Unstructured};
+use libfuzzer_sys::fuzz_target;
+use std::sync::atomic::{AtomicUsize, Ordering::SeqCst};
+use wasmtime_fuzzing::{generators, oracles};
+
+// Keep track of how many WebAssembly modules we actually executed (i.e. ran to
+// completion) versus how many were tried.
+static TRIED: AtomicUsize = AtomicUsize::new(0);
+static EXECUTED: AtomicUsize = AtomicUsize::new(0);
+
+fuzz_target!(|data: &[u8]| {
+    // errors in `run` have to do with not enough input in `data`, which we
+    // ignore here since it doesn't affect how we'd like to fuzz.
+    drop(run(data));
+});
+
+fn run(data: &[u8]) -> Result<()> {
+    let mut u = Unstructured::new(data);
+    let mut config: generators::Config = u.arbitrary()?;
+    config.module_config.set_differential_config();
+
+    // Enable features that the spec interpreter has implemented
+    config.module_config.config.bulk_memory_enabled = false;
+    config.module_config.config.reference_types_enabled = false;
+    config.module_config.config.simd_enabled = false;
+    config.module_config.config.relaxed_simd_enabled = false;
+    config.module_config.config.exceptions_enabled = false;
+    config.module_config.config.module_linking_enabled = false;
+    config.module_config.config.multi_value_enabled = false;
+
+    // TODO: seems like this should be implied by reference_types_enabled = false
+    config.module_config.config.max_tables = 1;
+
+    // TODO: this is a best-effort attempt to avoid errors caused by the
+    //       generated module exporting no functions.
+    config.module_config.config.min_exports = 5;
+    config.module_config.config.max_exports = 5;
+
+    let mut module = config.module_config.generate(&mut u)?;
+    module.ensure_termination(1000);
```

Comment:
Would it be possible to tackle this as a follow-up? If I'm interpreting correctly, currently the fuzzing config allows me to set `consume_fuel` to true, but automatically adds `u64:max_value()` fuel in this case. If we add the capability to configure the precise amount of fuel to be added via a parameter, it might be desirable to align other parts of the fuzzing infrastructure to use this parameter too (I'm looking at `generate_timeout` in `generators.rs`).

Reply:
Certainly! This can definitely be handled later.